### DISCUSSION TITLE
Addressing "Aw, Snap: Error code 11" Chrome crashes on Mac

### DISCUSSION THREAD

**Discussion by jimkyndemeyer** [pixijs/pixijs#7544] [Q&A] [answered]
Hi all,

I'm seeing the latest Chrome 91 crash on Mac when using Pixi 6 in a large project. If I temporarily don't add the Pixi root element to the DOM, the app works just fine for the other functionality, so the cause is a combination of Pixi and Chrome for Mac.

The same application works without issues on Windows.

It's Chrome error code 11, but since dev tools immediately disconnect I don't have anything to go on in terms of cause or prevention. The Chrome debug log is also left pretty much blank with no explanation.

Any suggestions or good strategies on how to address this?

It's a large project, so it would be quite difficult to figure out a minimal reproduction example without any error info to go on.

Thanks.

**Comment by ivanpopelyshev** [comment]
Search https://bugs.chromium.org/p/chromium/issues/list?q=webgl%20error%20code&can=2 , maybe this bug already is there

**Reply by jimkyndemeyer** [reply]
Thanks for the suggestion. There's more open bugs than I would have imagined, so I guess I'll give it a few weeks and then retest with a newer Chrome.

**Comment by jimkyndemeyer** [comment, marked as answer]
Leaving this here in case other devs run into similar errors: Turns out that this had to with text measurement and rendering of the Segoe UI font from Microsoft, but it only affected a Chrome on a 2015 MacBook Air. Switching to Open Sans on that device worked around the crash.

**Comment by MattTreichelYeah** [comment]
@jimkyndemeyer You wouldn't have any idea if whatever scenario you had also crashed in Safari, do you? I'm having a weird confluence of issues with the same "Error code 11", dev tools disconnect in Chrome, and also a crash in Safari where `context.measureText(metricsString).width` returns 0 and breaks PIXI.

**Reply by jimkyndemeyer** [reply]
Unfortunately never tested in Safari. I ended up with using localStorage to be able to override font choice on that single ageing MacBook back when I was still using it. The weird thing is that a newer MacBook never had the issue with the exact same version of Chrome.
